src/Map.js

```diff
@@ -0,0 +1,80 @@
+/* global L, Tangram */
+import React from 'react'
+import './Map.css'
+
+export default class Map extends React.Component {
+  componentDidMount () {
+    var INITIAL_VIEW = [56.46, 18.93]
```

Comment:
should this (and the below) be const?

Related: does reactapp come with a linter set up?

Reply:
Yep, but we'd have to rewrite much of the map code anyway in React (#22).

Re: linter; it does not, we'd have to bring our own. In this case, eslint (#16).

Reply:
I take that back. `eslint` is built in! We would need to bring our own configuration files though.